R/drop_dir.R

```diff
@@ -1,102 +1,126 @@
-
-#' drop_dir
-#'
-#'Retrieves file and folder metadata.
+#' List folder contents and associated metadata
 #'
-#' A sample output for contents returns the following fields:
-#' size, rev, thumb_exists, bytes, modified, client_mtime, path, is_dir, icon, root, mime_type, revision. Note that
-#' with verbose set to FALSE you only get a subset of these fields. To get the full output, look in \code{drop_dir()$contents}
-#' @template path
-#' @template file_limit
-#' @param  hash  Each call to /metadata on a folder will return a hash field, generated by hashing all of the metadata contained in that response. On later calls to /metadata, you should provide that value via this parameter so that if nothing has changed, the response will be a 304 (Not Modified) status code instead of the full, potentially very large, folder listing. This parameter is ignored if the specified path is associated with a file or if list=false.
-#' @param  list  The strings true and false are valid values. true is the default. If true, the folder's metadata will include a contents field with a list of metadata entries for the contents of the folder. If false, the contents field will be omitted.
-#' @param  include_deleted Only applicable when list is set. If this parameter is set to true, then contents will include the metadata of deleted children. Note that the target of the metadata call is always returned even when it has been deleted (with is_deleted set to true) regardless of this flag.
-#' @param  rev If you include a particular revision number, then only the metadata for that revision will be returned.
-#' @template locale
-#' @param include_media_info   If true, each file will include a photo_info dictionary for photos and a video_info dictionary for videos with additional media info. If the data isn't available yet, the string pending will be returned instead of a dictionary.
-#' @template include_membership
-#' @template verbose
+#' @param path path to folder in Dropbox to list contents of. Defaults to the root directory.
+#' @param recursive If true, the list folder operation will be applied recursively to all subfolders and the response will contain contents of all subfolders. Defaults to FALSE.
+#' @param include_media_info If true, FileMetadata.media_info is set for photo and video. Defaults to FALSE.
```

Comment:
For this and the next few lines of doc, do you want to use:

If `TRUE`

Just seems like a consistent thing to do as far as making sure even a cursory read should make it clear that it's a logical.  Not a mandatory change.

Reply:
done!